### ISSUE TITLE
Learn Storybook HTML

### ISSUE THREAD

**Issue by AstiV** [chromaui/learnstorybook.com#127]
Hey there,

not sure if this is the right way to approach you with the subject, so feel free to close this issue, if not.
First of all, thank you for providing this great resource. Did the tutorial for react and really enjoyed it.
Anyhow, I was wondering, if you are planning to add Storybook HTML (in combination with a templating system like mustache) to your curriculum any time soon?
We're considering using it at work but learning resources are quite rare...

Best wishes!
Astrid

**Comment by kylesuss** [comment]
Hi @AstiV 👋 

We don't have any current plans for this, though would welcome a community contribution for it if it is useful to enough people!

**Comment by AstiV** [comment]
Hi @kylesuss :)

Thanks for your reply! I see... Well, if I understand it well enough to be able to contribute, I am more than happy to do so. But first, I am trying to learn how to use it (and evaluate wether we implement it) 😅

**Comment by kylesuss** [comment]
Sure, I understand. Is the official guide helpful? 

https://storybook.js.org/docs/guides/guide-html/

Are your issues more about setup? Or the process of building out the stories?

**Comment by AstiV** [comment]
Oh dear, 

sorry I totally missed your reply @kylesuss. Thanks for getting back!
Yes, the link you provided, was the first thing I looked at to understand how to setup the project.

What I was looking for was a more detailed tutorial-like step-by-step format about both setup and the process of building out the stories...

I found this resource
http://derhess.de/2019/04/19/storybook-html-tutorial-part-1-project-setup/

But as soon as it comes to integrating a templating system, webpack steps into the game and I've had a hard time with configuration and understanding what to do in general.
This is why I did your react tutorial half way through this one.
